cypress/e2e/tests/migration/controls/businessservices/pagination.test.ts

```diff
@@ -81,7 +82,7 @@ describe(["@tier3"], "Business services pagination validations", function () {
         cy.wait(2 * SEC);
         goToLastPage();
         // Delete all items of last page
-        deleteTableRows();
+        deleteAllItems();
```

Comment:
I guess we agreed not to remove everything, but only things created by test

Reply:
In pagination tests some items are deleted for validation and then we don't know what to delete . In such cases we need to remove all Items.